src/mod/templates/main_index.html

```diff
@@ -67,6 +72,86 @@ <h2><a href="{{ app.subpage }}">{{ app.metadata.title }}</a>
                 <p class='details'><a href="{{ app.subpage }}">Show app details</a></p>
             </div>
         </div>
+
+        {% endif %}
+
+        {#
+            {% if not loop.last %}
+            <hr>
+            {% endif %}
+        #}
+        {% endfor %}
+    </div>
+
+    <!-- I don't know how to make this not a boilerplate code -->
+    <!-- The following is the same repeated block as above but for closed source apps -->
+
+    <h1>
+        Industrial Collaborations
+    </h1>
+    <small><i>Closed source apps</i></small>
+
+    <div id='entrylist'>
+        {% for app_name, app in apps|dictsort %}
+
+            <!-- Skip if its a closed source industrial partener -->
+            <!-- These apps will be displayed at the bottom -->
+            {% if app.metadata.industrial_collaboration %}
+
+        <div class="submenu-entry">
```

Comment:
If <div class="submenu-entry">...</div> is used twice, for maintainability reason, I would suggest to move it to its own file and then include the template twice in this file using the Jinja2 {% include %} directive. See, e.g., [here](https://stackoverflow.com/questions/22860085/how-do-i-include-a-html-file-in-a-jinja2-template).